webrtc.html

```diff
@@ -2638,40 +2619,45 @@ <h2>Methods</h2>
                     <a data-link-for="exception" data-lt="create">created</a>
                     <code>InvalidStateError</code>.</p>
                   </li>
+                  <li>
+                    <p>Return the result of <a href=
+                    "#dfn-chain-an-operation">chaining</a> the result of
+                    <a href="#dfn-create-an-offer">creating an offer</a> with
+                    <var>connection</var> to <var>connection</var>'s operations
+                    chain.</p>
+                  </li>
+                </ol>
+                <p>To <dfn>create an offer</dfn> given <var>connection</var>
+                run the following steps:</p>
+                <ol>
+                  <li>
+                    <p>If <var>connection</var>'s <a>signaling state</a> is
+                    neither <code>"stable"</code> nor
+                    <code>"have-local-offer"</code>, return a promise
+                    <a>rejected</a> with a newly <a data-link-for="exception"
+                      data-lt="create">created</a>
+                    <code>InvalidStateError</code>.</p>
+                  </li>
                   <li>
                     <p>If <var>connection</var> is configured with an identity
-                    provider, then begin <a data-cite="!WEBRTC-IDENTITY#sec.identity-verify-assertion">the identity
-                    assertion request process</a> if it has not already
-                    begun.</p>
+                    provider, then begin <a
+                    data-cite="!WEBRTC-IDENTITY#sec.identity-verify-assertion">
+                    the identity assertion request process</a> if it has not
+                    already begun.</p>
```

Comment:
In both createOffer and createAnswer: Shouldn't the identity assertion request process begin as part of the refactored algorithm? I think we need a part that includes both this bit and the "in parallel, begin the in-parallel steps..."
Otherwise setLocalDescription() is a way to bypass identity assertion.

Reply:
Although _7. Verifying Identity Assertions_ says "Identity assertion validation happens when setRemoteDescription is invoked on RTCPeerConnection.", I have a feeling this all should to be clarified, but for now I don't want to introduce any accidental changes in this PR.

Reply:
Note you're commenting on a word-wrap change. The surrounding changes, however, move this step from the outer *createOffer* steps to inside the new [**create an offer**](https://rawgit.com/jan-ivar/webrtc-pc/glareproof/webrtc.html#dfn-create-an-offer) algorithm, *specifically* so *setLocalDescription()* won't bypass it.

This means identity initialization may happen a tad later in some esoteric edge cases on the answerer side. e.g:
```js
await Promise.all([pc.setRemoteDescription(offer), pc.createAnswer()]);
```
but I think that's ok.

The " the identity assertion request process" is asynchronous, and is merely triggered to start here, not end.

Reply:
Ack